src/actions.js

```diff
@@ -329,6 +329,38 @@ export function getAllPosts(index) {
             });
     }
 }
+export function getUsersPosts(index, _id) {
+    let token = localStorage.getItem('token');
+
+    if (!token) {
+        return dispatch => {
+
+        }
+    }
+
+    return dispatch => {
+        return fetch(URLS.GET_USERS_POSTS, {
```

Comment:
I saw this or similar thing in the previous PR... Do not store XHR requests here in the action.